boards/openmote-cc2538/include/periph_conf.h

```diff
@@ -214,6 +221,17 @@ static const periph_spi_conf_t spi_config[SPI_NUMOF] = {
 #define GPIO_31_PIN         GPIO_PD7
 /** @} */
 
+/**
+ * @name OpenMote Pin to ADC Line Mapping
+ * @{
+ */
+#define AD4_PIN             2
+#define CTS_DI07_PIN        3
```

Comment:
These macros were added to help map the ADC lines to the actual board pins. Would it make sense to keep this, or is it a waste? It could be documented instead.